deploy-board/deploy_board/templates/deploys/deploys.tmpl

```diff
@@ -40,8 +41,9 @@
                 {{ deploy_summary.build|branchAndCommit }}
             </a>
         </td>
+        <td><a href="/clouds/accounts/{{ deploy_summary.account.provider }}/{{ deploy_summary.account.cell }}/{{ deploy_summary.account.id }}">{{ deploy_summary.account.id }}</a></td>
```

Comment:
The same here is in the previous comment - let's change how link looks like and add AWS account number with account name.